### DISCUSSION TITLE
CalendarView Show Selected Date Row

### DISCUSSION THREAD

**Discussion by Kiriakos41** [ME-MarvinE/XCalendar#52] [Q&A] [answered]
Here Default select is 20
![image](https://user-images.githubusercontent.com/70096169/174607777-f49774f9-a6ae-49ee-a670-1c24877e959d.png)
but to see the day i have to scroll down.
![image](https://user-images.githubusercontent.com/70096169/174607815-205702d3-e572-4112-bd43-43bd755f4aac.png)
Is any option that goes automatically to selected date?

**Comment by ME-MarvinE** [comment]
There is no way to do this currently. The CalendarView would need to expose the name of or the methods of the CollectionView it uses to display the dates which it does not do.

This may be possible in 4.0 once it is released.

**Comment by ME-MarvinE** [comment, marked as answer]
You can set the `DaysTemplate` of the CalendarView to a CollectionView with its ItemsSource bound to the `Days` of your Calendar.

After that you can reference the CollectionView like you would any other and use its 'ScrollTo' method for example.
